pom.xml

```diff
@@ -613,7 +613,7 @@
               <arg>-language:_</arg>
               <arg>-Xlint</arg>
               <arg>-Xfuture</arg>
-              <arg>-Xfatal-warnings</arg>
+              <!-- arg>-Xfatal-warnings</arg -->
```

Comment:
Did you try to solve the deprecated warnings by aliasing the types in class `Suppress.DeprecatedWarning`. I try to avoid disabling this argument.